packages/@uppy/provider-views/src/style/uppy-ProviderBrowser-viewType--list.scss

```diff
@@ -0,0 +1,63 @@
+// ***
+// View type: list
+// ***
+
+.uppy-ProviderBrowser-viewType--list {
+  background-color: $white;
+
+  li.uppy-ProviderBrowserItem {
+    display: flex;
+    align-items: center;
+    padding: 7px 15px;
+    margin: 0;
+
+    // Checkbox
+    > .uppy-ProviderBrowserItem-fakeCheckbox {
+      margin-right: 15px;
```

Comment:
Is deep-deep nesting like this necessary here? Ideally I’d lift a level or so, it becomes a little hard to follow and too specific, I think.

Reply:
@arturi, it isn't, it's just my personal preference, I think:
- it's easier to follow (because we can visualise the html hierarchy by reading css), and
- nesting is good (for the same reasons that we like to only have our Js variables accessible from a scope as little as possible).

But I understand it's an unusual preference, and in a library tight specificity may become a problem, I can unnest it.